backend/api/admin/src/main/java/co/airy/core/api/admin/Stores.java

```diff
@@ -79,6 +88,36 @@ public void onApplicationEvent(ApplicationStartedEvent event) {
 
         builder.<String, Template>table(applicationCommunicationTemplates, Materialized.as(templatesStore));
 
+        final KTable<String, User> usersTable = builder.table(applicationCommunicationUsers, Materialized.as(usersStore));
+
+        // Extract users from the op log to the users topic
+        builder.<String, HttpLog>stream(opsApplicationLogs)
+                .filter((logId, log) -> log.getUserId() != null)
+                .selectKey((logId, log) -> log.getUserId())
+                // Extract the Kafka record timestamp header
+                .transform(timestampExtractor())
```

Comment:
Transformers allow us to access the streams API on a lower level using the [Processor API](https://kafka.apache.org/10/documentation/streams/developer-guide/processor-api.html)

Reply:
Nice I will have a look